packages/devtools_app/lib/src/ui/icons.dart

```diff
@@ -53,24 +53,30 @@ class CustomIcon extends StatelessWidget {
 }
 
 class CustomIconMaker {
-  final Map<String, CustomIcon> iconCache = {};
+  final Map<String, Widget> iconCache = {};
 
-  CustomIcon getCustomIcon(String fromText,
+  Widget getCustomIcon(String fromText,
       {IconKind kind, bool isAbstract = false}) {
```

Comment:
not your fault, but trailing comma after `isAbstract = false`

Reply:
fixed